spec/loaders/doppler_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Anyway::Loaders::Doppler do
+  include Anyway::Testing::Helpers
+
+  subject { described_class.call }
+
+  let(:doppler_content) { { "some_content" => "success", "some_variable" => "variable" } }
+
+  context "when success loads data from doppler" do
+    it "loads data from doppler" do
+      http_success = Net::HTTPSuccess.new(1.0, '200', 'OK')
+
+      allow(http_success).to receive(:read_body) { doppler_content.to_json  }
+      expect_any_instance_of(Net::HTTP).to receive(:request) { http_success }
```

Comment:
Let's use `webmock` instead; we don't need to care about the underlying HTTP implementation here; it's likely to change over time (e.g., if we decide to add retry mechanism)

Reply:
Done

I included it in spec_helper and disabled all net connections

```
WebMock.disable_net_connect!(allow_localhost: true)
``